### ISSUE TITLE
Is there any way to re-check my previously "checked" manga chapters?

### ISSUE THREAD

**Issue by gooreat** [fmd-project-team/FMD#685]
Not really an issue and I'm not sure where to ask about it here. But here we go:

So yesterday I used FMD (after a long time) to check for the latest manga chapters and it works without problem. So I paused the tasks to resume the download tomorrow (which is today).

But instead of doing that, I went ahead and (stupidly) deleted all the tasks before it finishes downloading (because I want to refresh the new tasks with any updated chapters today). Then I check for new chapters, but FMD didn't capture my yesterday's chapters, and only got few update chapters. Unlike yesterday where most of them are blue and have lots of chapters for some of the tasks.

Is there any way to reset it and make FMD to search for my yesterday's chapters (including today)? I'm not sure If I have the time to look into my 700+ manga :P. Thanks

**Comment by Tmp341** [comment]
You need to manually edit your ``favorites.db`` and/or ``downloadedchapters.db`` in ``works`` folder.

**Comment by gooreat** [comment]
Oh edit it using DB softwares? or Notepad++?

Sorry lol Not very familiar with this stuff

Tried to open downloadedchapters.db with Notepad, and saw my manga entries there. Guess I just need to delete the latest entries/chapters? I don't think there is a way to check which manga that includes yesterday/new chapters, right? or there is?

Welp, guess I just have to do this manually from top to the bottom of the list lmao

**Comment by Tmp341** [comment]
> 
> 
> Oh edit it using DB softwares? or Notepad++?
> 
> Sorry lol Not very familiar with this stuff
> 
> Tried to open downloadedchapters.db with Notepad, and saw my manga entries there. Guess I just need to delete the latest entries/chapters? I don't think there is a way to check which manga that includes yesterday/new chapters, right? or there is?
> 
> Welp, guess I just have to do this manually from top to the bottom of the list lmao

Not notepad. DB Softwares, like ``SQLite Database Browser``

**Comment by traitorAIZEN** [comment]
i think you just have to check the website itself for what titles in your bookmarks have been updated recently

**Comment by SDXC** [comment]
sadly there is no other way than doing it manually or recovering a backup if you have any.

By default when deleting entries in the downloads tab you should be asked to confirm the deletion.
Did you disable this confirmation dialog and accidentally pressed DEL or something like that? If so, you should probably enable that confirmation dialog.

**Comment by Tmp341** [comment]
I'd written a batch file for myself, which backs up ``.db`` files in ``works`` folder (just creates a copy with ``.db.bak`` extension. It is a little late, but you can use it:

[BackupFav.zip](https://github.com/fmd-project-team/FMD/files/3969853/BackupFav.zip)

Just extract zip file inside ``works`` folder and when you wanna create backup files, just doubleclick on it.

**Comment by gooreat** [comment]
Sorry for the late reply. Been busy in the last few days.

Using the DB software does work, but I just did it manually (old-school way, checking each manga chapters, then download) in the end. Feel much faster than using software, at least for me lol

@SDXC Nope I got the confirmation, but still end up deleting them anyway because I want to refresh the tasks with new chapters and also because I thought it would work lmao (stupid of me but I learned my lesson :P)

@Tmp341 Thanks. Guess I'll use it everytime before I check for new chapters. lol

Just a suggestion, maybe (if it's possible and doesn't take long time or anything) implement auto database backup before checking for new chapters, etc?
